imsv-docs-astro/src/models/FundingProtocol.mjs

```diff
@@ -0,0 +1,50 @@
+import path from 'node:path';
+import { DeployedFundingProtocol } from './DeployedFundingProtocol.mjs';
```

Comment:
Where is this referenced? I see the type in a comment. Does this import need to be here?

Reply:
Yes, JSDoc needs the import.